Wrappers.cpp

```diff
@@ -0,0 +1,59 @@
+#include "Wrappers.h"
+#include "Sailboat.h"
+#include "SchoolDay.h"
+#include "SurfReport.h"
+#include "SchoolWeek.h"
+#include "Lineup.h"
+
+SailboatWrapper::SailboatWrapper( Sailboat* ptr ) : pointerToSailboat( ptr ) {}
+
+SailboatWrapper::~SailboatWrapper()
+{
+    delete pointerToSailboat;
+}
+
+MastWrapper::MastWrapper( Mast* ptr ) : pointerToMast( ptr ) {}
+
+MastWrapper::~MastWrapper()
+{
+    delete pointerToMast;
+}
```

Comment:
where is `Mast` defined?
you need to include that file where the type is being used.
- [ ] revise